getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
+
+- **Timestamp**: When the event occurred.
+- **Attributes**: A set of key-value pairs offering details about the event context.
+- **Metadata**: Contextual labels and IDs, such as trace ID and span ID, that connect related events.
+
+Rather than simply calling them logs or metrics, event data reflects a broader range of interactions, crossing boundaries from engineering to product management, security, and beyond. This includes everything from the number of documents created in a product to the sequence of API calls during a checkout process.
+
+## Types of event data
+
+Traditionally, observability has been associated with three pillars, each effectively a specialized view of event data:
+
+- **Logs**: Logs record discrete events, such as error messages or access requests, typically associated with engineering or security.
+- **Traces**: Traces track the path of requests through a system, capturing each step’s duration. By linking related spans within a trace, developers can identify bottlenecks and dependencies.
+- **Metrics**: Metrics quantify state over time, recording data like CPU usage or user count at intervals. Product or engineering teams can then monitor and aggregate these values for performance insights.
+
+In Axiom, these observability elements are stored as event data, allowing for fine-grained, efficient tracking across all three pillars.
+
+## Metrics support
```

Comment:
Putting all of the above to one side, yes, this is good. It answers the question of our current metrics status, which is not well covered elsewhere.